_deployment/tests/twitter.sh

```diff
@@ -4,26 +4,29 @@
 check_twitter () {
   handles="$(git diff origin/master...HEAD ../../_data | grep "^+[[:space:]]*twitter:" | cut -c15-)"
 
-  if [ -z "$handles" ]; then
+  if [[ ! "$handles" =~ [^[:space:]] ]]; then
```

Comment:
While having a space at the end of the value of the `twitter` tag doesn't break anything, it also doesn't improve anything.
As I tried to weigh up the pros and cons I can't really find any pros for it, and a con would be that it adds unnecessary data to the file (yes, it's a bit extreme but I'm just laying everything on the table here).

Reply:
Currently, an extra space before or after the Twitter/Facebook handle _does_ make the check fail, which is one of the reasons for this PR. You can't run the Twitter check in a local installation, since the API key is not public. Also, the extra spaces are not visible most of the time (some editors display them differently), and they are perfectly valid YAML syntax. So even though additional spaces are rather useless, they should not lead to a failed check.